src/main/java/org/folio/rest/impl/BatchVoucherExportConfigHelper.java

```diff
@@ -65,7 +68,9 @@ public Future<Void> putExportConfig(ExportConfig exportConfig) {
 
   public Future<Void> putExportConfigCredentials(String id, Credentials credentials) {
     String path = String.format(resourcesPath(BATCH_VOUCHER_EXPORT_CONFIGS_CREDENTIALS), id);
-    return restClient.put(path, credentials, buildRequestContext());
+    return StringUtils.isBlank(credentials.getUsername()) || StringUtils.isBlank(credentials.getPassword()) ?
```

Comment:
we need some logs in these 2 places in this class, because this logic is not so obvious